src/resources/views/crud/fields/upload.blade.php

```diff
@@ -1,11 +1,7 @@
 @php
-   if (!isset($field['wrapper']) || !isset($field['wrapper']['data-init-function'])){
-        $field['wrapper']['data-init-function'] = 'bpFieldInitUploadElement';
-    }
-
-    if (!isset($field['wrapper']) || !isset($field['wrapper']['data-field-name'])) {
-        $field['wrapper']['data-field-name'] = $field['name'];
-    }
+    $field['wrapper'] = $field['wrapper'] ?? $field['wrapperAttributes'] ?? [];
+    $field['wrapper']['data-init-function'] = 'bpFieldInitUploadElement';
```

Comment:
I think to make this non-BC we need here `$field['wrapper']['data-init-function'] = $field['wrapper']['data-init-function'] ?? 'bpFieldInitUploadElement';`

The same for the `$field['wrapper']['data-field-name']`

I agree that the init function should be ours, but if in code it allowed to be replaced by developer own custom init function, we should continue respecting that, atleast until 4.2.

Reply:
@pxpm I agree, I think this was a mistake.
I'll do the changes accordingly your suggestions.